### ISSUE TITLE
Contrast Issue with search relevance links

### ISSUE THREAD

**Issue by ayanchoudhary** [cc-archive/vocabulary-legacy#477]
## Problem

![CC Search - Brave 17-06-2020 00_03_55](https://user-images.githubusercontent.com/45792768/84813981-68b68e00-b02e-11ea-8251-42882867a238.png)

## Description

Currently, this span has a contrast ratio of **2.23** whereas the desired minimum contrast ratio should be **4.5** or **AA/AAA large**

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by ayanchoudhary** [comment]
@panchovm @dhruvkb

**Comment by zackkrida** [comment]
Closing in favor of https://github.com/creativecommons/cccatalog-frontend/issues/1221 on CC Search.
